projects/dendrites/README.md

```diff
@@ -67,4 +73,7 @@ The notebook `permutedMNIST/figures/hidden_activations_per_task.ipynb` generates
 
 ----
 
-Karan Grewal, Jeremy Forest, Benjamin P. Cohen, and Subutai Ahmad. “Going Beyond the Point Neuron: Active Dendrites and Sparse Representations for Continual Learning”. In: *bioRxiv*(2021). doi:10.1101/2021.10.25.465651. url:[https://www.biorxiv.org/content/early/2021/10/26/2021.10.25.465651](ttps://www.biorxiv.org/content/10.1101/2021.10.25.465651v1).
+Karan Grewal, Jeremy Forest, Benjamin P. Cohen, and Subutai Ahmad. “Going Beyond the Point Neuron: Active Dendrites and Sparse Representations for Continual Learning”. In: *bioRxiv*(2021). doi:10.1101/2021.10.25.465651. url:[https://www.biorxiv.org/content/early/2021/10/26/2021.10.25.465651](https://www.biorxiv.org/content/10.1101/2021.10.25.465651v1).
```

Comment:
I think we should use https://www.biorxiv.org/content/early/2021/10/26/2021.10.25.465651, not the URL with v1 added. The v1 link will be obsolete once we respond to reviewer comment.s